src/planning.py

```diff
@@ -240,6 +254,10 @@ def _skeleton_generator(
     rng_prio = np.random.default_rng(seed)
     hq.heappush(queue,
                 (heuristic(root_node.atoms), rng_prio.uniform(), root_node))
+    # Initialize with empty skeleton for root
```

Comment:
sorry to be annoying about this, but please make the format of all your comments consistent :) This one should have a period at the end, and there's one below missing a space after the hashtag